src/scenes/Lines/scenes/Editor/JourneyPatterns/Editor/ServiceJourneys/Editor/DayTypeEditor/index.tsx

```diff
@@ -1,36 +1,28 @@
 import React from 'react';
 import { useSelector } from 'react-redux';
 import { selectIntl } from 'i18n';
-import PropTypes from 'prop-types';
 import { Label } from '@entur/typography';
 import { DayType } from 'model';
 import DayTypeAssignmentsEditor from './DayTypeAssignments';
 import WeekdayPicker from 'components/WeekdayPicker';
 import { DAY_OF_WEEK } from 'model/enums';
 import './styles.scss';
 import messages from '../../../messages';
+import DayTypeAssignment from '../../../../../../../../../model/DayTypeAssignment';
```

Comment:
Det skal gå att bara importera denna med en relative path från root.
```suggestion
import DayTypeAssignment from 'model/DayTypeAssignment';
```

Reply:
Går det faktisk? Trodde TypeScript klagde da.

Reply:
Inte i min vscode iallafall 🤓